alphastats/gui/pages/04_Analysis.py

```diff
@@ -1,12 +1,12 @@
 import streamlit as st
+from gui.utils.options import get_plotting_options, get_statistic_options
```

Comment:
I get that we wanted to declutter the session state. How expensive is it to run these functions? Maybe we can at least restrict that to once per page reload, or cache the result.

Reply:
they will be removed in a later PR